autovalet_lane_detection/src/autovalet_lane_detection/LaneDetector.py

```diff
@@ -78,12 +89,35 @@ def detectLaneRGBD(self, color_img, depth_img):
             publishCloud(ego_line, self.camera.header.frame_id, self.egoLine_pub)
             # maybe publish center line alone too?
 
-            return lane_cloud, ego_line
+            ## Parking directions
+            if  center_line_coordinates.shape[0] % 2 == 0:
+                index = center_line_coordinates.shape[0] / 2 
+            else:
+                index = (center_line_coordinates.shape[0] + 1)/ 2
+            center_line_coordinates= center_line_coordinates[index,:]
+            
+            if center_line_coordinates.shape[0] != 0:
+                center_line_cloud = self.line2cloud(depth_img, center_line_coordinates.reshape(1,-1))    # px3      
+
+            transf = self.tf_buffer.lookup_transform(self.target_id, # target_frame_id
+                                        self.source_id, # source frame
+                                        rospy.Time(0), # get the tf at first available time
+                                        rospy.Duration(1.0)) # timeout after 1
```

Comment:
I'm being OCD but can you align the arguments plz 🙏

Reply:
lol done